docs/generate_example_images.py

```diff
@@ -11,7 +11,8 @@
 mod = importlib.import_module("kornia.augmentation")
 
 BASE_IMAGE_URL: str = "https://raw.githubusercontent.com/kornia/data/main/panda.jpg"
-OUTPUT_PATH = Path(__file__).absolute().parent / "source/_static"
+OUTPUT_PATH = Path(__file__).absolute().parent / "build/html/_static"
```

Comment:
This should be irrelevant to this PR.

Reply:
Good catch, no idea, how it ended up here...